collection-scripts/gather_network_logs

```diff
@@ -4,18 +4,31 @@ NETWORK_LOG_PATH="${BASE_COLLECTION_PATH}/network_logs/"
 
 mkdir -p ${NETWORK_LOG_PATH}/
 
-WORKERS=$(oc get nodes --no-headers -o custom-columns=":metadata.name")
+NODES=$(oc get nodes --no-headers -o custom-columns=":metadata.name")
+PIDS=()
 
-for WORKER in $WORKERS; do
-    SDN_POD=$(oc -n openshift-sdn get pods --no-headers -o custom-columns=":metadata.name" --field-selector spec.nodeName=$WORKER -l app=sdn)
-    oc -n openshift-sdn exec $SDN_POD -- bash -c "iptables-save -c" > ${NETWORK_LOG_PATH}/${WORKER}_iptables
-    oc -n openshift-sdn exec $SDN_POD -- bash -c "ovs-vsctl show" > ${NETWORK_LOG_PATH}/${WORKER}_ovs_dump
-    oc -n openshift-sdn logs $SDN_POD > ${NETWORK_LOG_PATH}/${SDN_POD}_sdn_logs
-done
+for NODE in $NODES; do
+    SDN_POD=$(oc -n openshift-sdn get pods --no-headers -o custom-columns=":metadata.name" --field-selector spec.nodeName=$NODE -l app=sdn)
+    oc -n openshift-sdn exec $SDN_POD -- bash -c "iptables-save -c" > ${NETWORK_LOG_PATH}/${NODE}_iptables &
+    PIDS+=($!)
+    oc -n openshift-sdn exec $SDN_POD -- bash -c "ovs-vsctl show" > ${NETWORK_LOG_PATH}/${NODE}_ovs_dump &
+    PIDS+=($!)
+    oc -n openshift-sdn logs $SDN_POD > ${NETWORK_LOG_PATH}/${SDN_POD}_sdn_log &
+    PIDS+=($!)
 
-MASTERS=$(oc get nodes --no-headers -o custom-columns=":metadata.name" -l node-role.kubernetes.io/master)
+    OVS_POD=$(oc -n openshift-sdn get pods --no-headers -o custom-columns=":metadata.name" --field-selector spec.nodeName=$NODE -l app=ovs)
+    oc -n openshift-sdn exec $OVS_POD -- bash -c "cat /var/log/openvswitch/ovs-vswitchd.log" > ${NETWORK_LOG_PATH}/${OVS_POD}_vswitchd_log &
+    PIDS+=($!)
+    oc -n openshift-sdn exec $OVS_POD -- bash -c "cat /var/log/openvswitch/ovsdb-server.log" > ${NETWORK_LOG_PATH}/${OVS_POD}_ovsdb_log &
+    PIDS+=($!)
 
-for MASTER in $MASTERS; do
-    CONTROLLER_POD=$(oc -n openshift-sdn get pods --no-headers -o custom-columns=":metadata.name" --field-selector spec.nodeName=$MASTER -l app=sdn-controller)
-    oc -n openshift-sdn logs $CONTROLLER_POD > ${NETWORK_LOG_PATH}/${CONTROLLER_POD}_controller_logs
+    CONTROLLER_POD=$(oc -n openshift-sdn get pods --no-headers -o custom-columns=":metadata.name" --field-selector spec.nodeName=$NODE -l app=sdn-controller)
+    if [ -n "$CONTROLLER_POD" ]; then
+       oc -n openshift-sdn logs $CONTROLLER_POD > ${NETWORK_LOG_PATH}/${CONTROLLER_POD}_controller_logs &
+       PIDS+=($!)
+    fi
```

Comment:
https://github.com/openshift/must-gather/pull/94/files/94623a418b7d7b733709245012286a17b67ade26..98cfda1aa47bc9d4cd72113c45e2a29f7c37157d#discussion_r289164403 still applies

Reply:
I'm not seeing this get collected in CI so we should keep this (ignore my original comment).

Reply:
I don't follow this comment, the link doesn't tell me what you want, can you pls clarify?

Reply:
I thought this was collected by a relationship in must-gather the binary (I'm validating that that is not correct), and that this collection is ok.